editors/code/src/client.ts

```diff
@@ -95,7 +95,9 @@ export async function createClient(
                     const resp = await next(params, token);
                     if (resp && Array.isArray(resp)) {
                         return resp.map((val) => {
-                            return substituteVSCodeVariables(val);
+                            return prepareVSCodeConfig(val, (key, cfg) => {
+                                cfg[key] = config.discoveredWorkspaces;
+                            });
```

Comment:
I don't quite follow this but doesn't this overwrite everything in the config?

Reply:
whoops, I think you're right about that. Thanks for the catch!